project/scalatest.scala

```diff
@@ -1448,7 +1449,7 @@ object ScalatestBuild {
         organization := "org.scalatest",
         moduleName := "scalatest-app",
         libraryDependencies ++= nativeCrossBuildLibraryDependencies.value,
-        libraryDependencies += "org.scala-native" %%% "test-interface" % "0.3.6",
+        libraryDependencies += "org.scala-native" %% "test-interface_native0.4.0-M2" % "0.4.0-M2",
```

Comment:
Same here:
```scala
libraryDependencies += "org.scala-native" %%% "test-interface" % nativeVersion
```

Reply:
@lolgab Adjustments made, thanks for pointing them out.